[ 2024-11-04 16:28:38,163 ] 1753 asyncio - ERROR - Exception in callback BaseAsyncIOLoop._handle_events(15, 1)
handle: <Handle BaseAsyncIOLoop._handle_events(15, 1)>
Traceback (most recent call last):
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/site-packages/tornado/platform/asyncio.py", line 202, in _handle_events
    handler_func(fileobj, events)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/site-packages/tornado/iostream.py", line 721, in _handle_events
    self.io_loop.update_handler(self.fileno(), self._state)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/site-packages/tornado/platform/asyncio.py", line 177, in update_handler
    self.selector_loop.remove_reader(fd)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/asyncio/selector_events.py", line 336, in remove_reader
    return self._remove_reader(fd)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/asyncio/selector_events.py", line 279, in _remove_reader
    self._selector.unregister(fd)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/selectors.py", line 539, in unregister
    self._selector.control([kev], 0, 0)
TypeError: changelist must be an iterable of select.kevent objects
[ 2024-11-04 16:28:39,348 ] 1753 asyncio - ERROR - Exception in callback BaseAsyncIOLoop._handle_events(12, 1)
handle: <Handle BaseAsyncIOLoop._handle_events(12, 1)>
Traceback (most recent call last):
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/site-packages/tornado/platform/asyncio.py", line 202, in _handle_events
    handler_func(fileobj, events)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/site-packages/tornado/iostream.py", line 721, in _handle_events
    self.io_loop.update_handler(self.fileno(), self._state)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/site-packages/tornado/platform/asyncio.py", line 177, in update_handler
    self.selector_loop.remove_reader(fd)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/asyncio/selector_events.py", line 336, in remove_reader
    return self._remove_reader(fd)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/asyncio/selector_events.py", line 279, in _remove_reader
    self._selector.unregister(fd)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/selectors.py", line 539, in unregister
    self._selector.control([kev], 0, 0)
TypeError: changelist must be an iterable of select.kevent objects
[ 2024-11-04 16:28:39,520 ] 1753 asyncio - ERROR - Exception in callback BaseAsyncIOLoop._handle_events(9, 1)
handle: <Handle BaseAsyncIOLoop._handle_events(9, 1)>
Traceback (most recent call last):
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/site-packages/tornado/platform/asyncio.py", line 202, in _handle_events
    handler_func(fileobj, events)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/site-packages/tornado/iostream.py", line 721, in _handle_events
    self.io_loop.update_handler(self.fileno(), self._state)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/site-packages/tornado/platform/asyncio.py", line 177, in update_handler
    self.selector_loop.remove_reader(fd)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/asyncio/selector_events.py", line 336, in remove_reader
    return self._remove_reader(fd)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/asyncio/selector_events.py", line 279, in _remove_reader
    self._selector.unregister(fd)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/selectors.py", line 539, in unregister
    self._selector.control([kev], 0, 0)
TypeError: changelist must be an iterable of select.kevent objects
[ 2024-11-04 16:28:39,527 ] 1753 asyncio - ERROR - Exception in callback BaseAsyncIOLoop._handle_events(17, 1)
handle: <Handle BaseAsyncIOLoop._handle_events(17, 1)>
Traceback (most recent call last):
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/site-packages/tornado/platform/asyncio.py", line 202, in _handle_events
    handler_func(fileobj, events)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/site-packages/tornado/iostream.py", line 721, in _handle_events
    self.io_loop.update_handler(self.fileno(), self._state)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/site-packages/tornado/platform/asyncio.py", line 177, in update_handler
    self.selector_loop.remove_reader(fd)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/asyncio/selector_events.py", line 336, in remove_reader
    return self._remove_reader(fd)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/asyncio/selector_events.py", line 279, in _remove_reader
    self._selector.unregister(fd)
  File "/Users/rajkumarrawal/Documents/GitHub/MCQGEN/env/lib/python3.9/selectors.py", line 539, in unregister
    self._selector.control([kev], 0, 0)
TypeError: changelist must be an iterable of select.kevent objects
